index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+        display: flex;
+        flex-direction: row;
+      }
+      .section{
+        display: flex;
+        flex-direction: column;
+      }
+      .navbar{
+        visibility: visible;
+        justify-content: flex-start;
+        flex-direction: column;
+      }
+      @media (max-width: 12450px) { 
+        body{
+          background-color: gray;
+        }
+        
+       }
+    </style>
+  </head>
+  
+  <body>
+    
+    <main id = "main-doc">
+      <div class="container">
+      <div class="navbar">
+        <nav id="navbar">
```

Comment:
It is better to use the <nav> tag instead of the <div> tag as it helps with SEO and better readability 👍🏾.

Reply:
- [ ] Good start. Please, note that you need to phrase the comments politely as suggestions instead of order. For example: "It is better to use the tag instead of the tag as it helps with SEO and better readability"